packages/traceability/issuer-irec-api-wrapper/src/Issue.ts

```diff
@@ -23,7 +23,7 @@ export class Issue extends FileIds {
     @Expose({ name: 'fuel_code', toPlainOnly: true })
     @IsString()
     @IsNotEmpty()
-    fuel: string;
+    deviceType: string;
```

Comment:
Mismatch between `@Expose({ name: 'fuel_code', toPlainOnly: true })` and `deviceType: string`.

Reply:
same as above